src/diffusers/modeling_utils.py

```diff
@@ -297,7 +298,12 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
         subfolder = kwargs.pop("subfolder", None)
         device_map = kwargs.pop("device_map", None)
 
-        user_agent = {"file_type": "model", "framework": "pytorch", "from_auto_class": from_auto_class}
+        user_agent = {
+            "diffusers": __version__,
+            "file_type": "model",
+            "framework": "pytorch",
+            "from_auto_class": from_auto_class,
```

Comment:
```suggestion
```
Think we don't really have the `from_auto_class` features yet (we should pass this via the `DiffusionPipeline` class eventually, but maybe in a next PR? :-))

Reply:
Ah, this got inherited when we were copying the model loaders from transformers. Should be removed indeed :)